src/context.cpp

```diff
@@ -21,6 +21,7 @@
 #include "environment.h"
 #include "context.h"
 #include "fw-update/fw-update-factory.h"
+#include "backend.h"
```

Comment:
According to the [Travis log](https://travis-ci.org/IntelRealSense/librealsense/jobs/574816158) you've changed the line ending from Linux (lf) to Win (crlf)
This needs to be addressed:
>[ERROR] src/context.cpp is using DOS line endings (this project is using Unix line-endings)
Pull-Request check failed, please address 1 the errors reported above